deploy/charts/approver-policy/templates/deployment.yaml

```diff
@@ -77,15 +77,15 @@ spec:
 
       hostNetwork: {{ .Values.app.webhook.hostNetwork }}
```

Comment:
@wallrj does it make sense that `hostNetwork` and `dnsPolicy` remain as options under the webhook value?

Reply:
Yeah, I noticed those. Probably not. I'll move those too.

Reply:
Done.